chunky/src/java/se/llbit/math/NodeBasedOctree.java

```diff
@@ -164,93 +164,98 @@ public boolean isInside(Vector3 o) {
     return lx == 0 && ly == 0 && lz == 0;
   }
 
+  // Moves the ray to the boundary of the octree. Returns false when the ray doesn't intersect the octree.
```

Comment:
This could be proper javadoc, maybe you could add that for all the octree classes while you're at it?